src/pickley/bstrap.py

```diff
@@ -135,7 +134,7 @@ def bootstrap_pickley_with_pip(self, venv_folder: Path):
         if not needs_virtualenv and not DRYRUN:
             needs_virtualenv = not is_executable(pip)
 
-        if needs_virtualenv:
+        if needs_virtualenv:  # pragma: no cover, not testing py3.6 fallback anymore
```

Comment:
`requires_python` seems to indicate you can work on 3.6, is that not true?

Reply:
Yes, here I would need to simulate a `python3 -mvenv ...` failing. Some distros for example do not provide the std lib module `venv`. Some very old python versions have a buggy `venv` module (IIRC).

Here, the intent is to double check on whether `-mvenv` actually did its job, and if it didn't, then download and call `python .../virtualenv.pyz ...` as a fallback.

This is known to work right now, but yeah wonder if I should keep testing this (as it requires a complex test setup), since this is not a thing anymore now for ages (`-mvenv` failing) + we don't even need `-mvenv` with uv.